edx_proctoring/views.py

```diff
@@ -1000,7 +1001,7 @@ def get(self, request, course_id, exam_id=None):
             backend = get_backend_provider(exam)
             if backend:
                 user = {
-                    'id': request.user.id,
+                    'id': obscured_user_id(request.user.id, exam['backend']),
```

Comment:
What's the purpose of including the backend in the hash?

Reply:
It’s an extra degree of paranoia — each backend will have a different user id for the same user. (It’s a bit of a moot point, considering we’re sending the user’s email address, as well...)